servermaster/jobmanager.go

```diff
@@ -69,6 +69,9 @@ func (jm *JobManagerImplV2) QueryJob(ctx context.Context, req *pb.QueryJobReques
 			}
 			return resp
 		}
+		if masterMeta != nil {
+			log.L().Warn("load master kv meta from meta store, but status is not expected", zap.Any("id", req.JobId), zap.Any("status", masterMeta.StatusCode))
+		}
```

Comment:
How to reproduce this unepxected case in test?

Reply:
After thinking twice, It seems this branch is impossible. If jobmaster offline normally, the meta should be set finished, or it needs failover and is in pending queue.

Reply:
How about keeping this log as an unharmful warn?

Reply:
Fine